### PR TITLE
Issue #16

### PR DESCRIPTION
**Pull request by dalSara**
I dont have anything more to add at the moment. Would be nice if someone else will look in to it and maybe add some of the missing info.

### CODE REVIEW

File: new-employees/Secrets-in-the-office.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* NO Nytt og nyttig fra HR og Admin. Norway
+<!--more groups?-->
+
+## Time registration ##
+All employees in Creua will have to endure registration of their work hours in Unit4. 
+The experience can be made better with this Chrome [plugin](https://github.com/Creuna-Oslo/Unit4Enhancements). 
+There is some good instruction videos in the workplace [Unit4 Group](https://creuna.facebook.com/groups/1056335684500691/videos/). The information for registrate your hours for diffrent projects is usually given by the project leader. 
+
+## Access cards and the alarm ##
+You have most likely got an access card and a code for that card. Remember that code. You will need that code if you arrive early (before 8.00)or work late(after 16.00).  After 23.00 and in the weekends the alarm is active in the office. If you use your card at the card reader the alarm will be off for 3 hours. If the alarm starts you will have to call Nokas on 02580 to deactivate it. You will have to give the password to Nokas. By obvious reasons I will not print the password here..
```
Comment (asbjornh) [inline new-employees/Secrets-in-the-office.md]: Should we have details about  alarm system times publicly available?

Comment (pellebjerkestrand) [reply new-employees/Secrets-in-the-office.md]: No, we should not.

Comment (dalSara) [reply new-employees/Secrets-in-the-office.md]: Added it on request. But now it is gone.

**Comment by asbjornh** [comment]
Here's a clickable thing to see the related issue: #16

**Comment by asbjornh** [comment]
I found several typos when reading through it, but I didn't have time to comment on them. If this gets merged before I find the time, we can just create a new PR for that.

**Comment by Reasonable-Solutions** [comment]
>  Would be nice if someone else will look in to it and maybe add some of the missing info.

We cannot see what is missing because it is not there. 

Merge this and we can  I T E R A T E.

**Pull request merged**